docs/reference/faq-troubleshooting.md

```diff
@@ -1,17 +1,75 @@
+extra (not cross-seeding) this is the best solution.
+
+Consider using infoHash if you are racing using data-based searches to prevent mismatches.
+
+### My data-based search is searching torrent files!
+
+Torrent files in torrentDir take precedence over data files with the same name and are deduplicated. If you need to search
+specifically the dataDir, use an empty directory for torrentDir temporarily. This will strictly search the data.
+
+### Error: ENOENT: no such file or directory
+
+This is usually caused by broken symlinks in your dataDir during a data-based search. Check the `/logs/verbose.\*.log` files for the file causing this.
+
+### What linkType should I use? (data-based searching)
+
+Your options are `hardlink` or `symlink`. These operate in seperate ways, and depending on your workflow you should choose appropriately. This is a brief description, however
+a more in depth guide is available at [Trash's Hardlinking Guide](https://trash-guides.info/Hardlinks/Hardlinks-and-Instant-Moves/).
+
+- symlinks are a "shortcut" of a sort, pointing at the original file from a new location. This can be used across mounts (docker) or partitions/drives. The issue is that if the original file is deleted when you remove a torrent, the torrents in your client using the symlink will "break" and you will receive errors. If this sounds like a hassle, consider reading further about hardlinks.
+
+- hardlinks are a tricky thing for someone not familiar, but are worth understanding. They are a "direct line" to the actual data on the disk and are indistinguishable from the original to the file system. They do not require any extra space. These linktypes are only possible on the same partition/disk and you will need to have your linkDir set to the same mount (docker) or partition. This is the best approach if you do not keep torrents in your client (due to them being deleted from the tracker) - which would then break symlinks and cross-seeds. This is the approach commonly used in Arr's on import.
+
+### Searches aren't working from VPN'd torrent client
+
+If you are running your searches on completion from a VPN'd torrent client, it is likely you will need to set up split tunneling. You will not be able to access your local instance (which should not be publicly exposed) from the VPN due to localhost/NAT not being accessible from the VPN. If you are running a VPN'd torrent client in docker, you can use your cross-seed's container IP or name (if using custom docker networks.)
+
+### Searching media libraries vs. torrent data (data-based searching)
+
+You can search both your media libraries (Arr/Plex) and actual torrent data (downloaded files.) If you are using the media libraries, you will need to use `matchMode: "risky"` in your configuration file to allow cross-seed some leeway in its matching process for these renamed files. `risky` `matchMode` is not recommended to be used without skipRecheck set to false, as this is not as accurate as safe.
```

Comment:
```suggestion
You can search both your media libraries (Arr/Plex) and actual torrent data (downloaded files). If you are using the media libraries with renamed files, you will need to use `matchMode: "risky"` in your configuration file to allow cross-seed some leeway in its matching process. `risky` `matchMode` is not recommended to be used without skipRecheck set to false, as it may result in more false positives than `safe`.
```